content/hi/docs/tutorials/kubernetes-basics/expose/expose-interactive.html

```diff
@@ -0,0 +1,40 @@
+---
+title: इंटरएक्टिव ट्यूटोरियल - अपने ऐप को उजागर करना
+weight: 20
+---
+
+<!DOCTYPE html>
+
+<html lang="en">
```

Comment:
```suggestion
<html lang="hi">
```

Reply:
as we are doing Hindi Localization!!
We always encourage to refer other localizations, as you would get a much better idea on what to change!!